src/components/MainTextOutput.js

```diff
@@ -26,6 +32,9 @@ export default (props) => {
               <Span2><p>{text.players.length === 1 ? `${text.players} is` : `${text.players.join(", ")} are`} standing here</p></Span2>
             )}
             {text.message && (<Span3><p>{text.message}</p></Span3>)}
```

Comment:
The formatting here could be slightly better, breaking this up on to separate lines would make things a bit easier to read and would prevent that weird lack of indenting on line 32